components/centraldashboard/test/e2e.test.ts

```diff
@@ -0,0 +1,144 @@
+import 'jasmine';
+
+import {Credentials, JWT} from 'google-auth-library';
+import {launch, Page} from 'puppeteer';
+import request from 'request';
+
+
+/**
+ * Look here to see setup instructions for Environment Setup for Cluster Auth:
+ *   https://cloud.google.com/kubernetes-engine/docs/tutorials/authenticating-to-cloud-platform
+ * 
+ * Use Scope: Owner
+ */ 
+const ORIGINAL_TIMEOUT = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+const {
+    KF_HOST,   // The url to your cluster, usually: https://<cluster>.endpoints.<project>.cloud.goog/
+    CLIENT_ID, // This is the CLIENT_ID token used to create the cluster (used in http://deploy.kubeflow.cloud/)
+    SERVICE_ACCOUNT_EMAIL, // The email address for the service account you created <name>@<project>.iam.gserviceaccount.com
+    SERVICE_ACCOUNT_KEY,   // Should be a JSON file downloaded from cloud console service accounts
+} = process.env;
+
+['KF_HOST', 'CLIENT_ID', 'SERVICE_ACCOUNT_EMAIL', 'SERVICE_ACCOUNT_KEY'].forEach(envVar => {
+    if (!process.env[envVar]) {console.log(`${envVar} environment variable must be set`);process.exit(1);}
+});
+if (!/^https?:\/\/\S+/.test(KF_HOST)) {console.log('Invalid HOST url provided, must be like http*://*');process.exit(1);}
```

Comment:
nit: `Invalid HOST url provided, must be like https*://*` (the protocol should be https in the end)

Reply:
Right, usually clusters redirect, so I allowed users to pass in `http` or `https`, but I've changed the log statement AND regex to simply only allow `https`